content/blog/2018/app-metrics-istio/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Application Metrics in Istio
+description: How to send application metrics to Prometheus instance in Istio.
+publishdate: 2018-12-19
+subtitle:
+attribution: Mete Atamel
+twitter: meteatamel
+weight: 78
+keywords: [metrics, prometheus]
```

Comment:
Please consider adding more keywords. Our system provides readers links at the end of your blog based on these keywords. At the very least, I would add: Mixer, Envoy, Sidecar, and Traffic Management.

Reply:
Added